src/Toast/README.md

```diff
@@ -80,4 +126,23 @@ notes: ''
     </>
   );
 }
+```
+
+### Theme variables (SCSS)
```

Comment:
Can be removed per https://github.com/edx/paragon/pull/931 (see examples in that PR)